### PR TITLE
change: uses centralized bookmarklet resource 🔖

### PR DESCRIPTION
**Pull request by jahorton**
One of three PRs fixing https://github.com/keymanapp/keyman/issues/4785.

Note that this relies upon https://github.com/keymanapp/r.keymanweb.com/pull/7, one of the other two PRs contributing to the fix.

### CODE REVIEW

File: cdn/dev/js/kmwlive.js [hunk 1]
```diff
@@ -443,24 +443,15 @@ function updateBookmarklet(kbd) {
...
-      function() {
-        if(typeof _gaq != 'undefined')
-          _gaq.push(['_trackEvent', 'Bookmarklet', 'Installing', kbd.LanguageCode+','+kbd.Name ]);
-      });
+    let bml_element_old = document.getElementsByClassName('keyman-bookmarklet').item(0);
+    let bml_parent = bml_element_old.parentElement;
+
+    let kbdid = kbd.InternalName.substr('Keyboard_'.length);
+    let label = kbd.LanguageName + ' Keyboard';
+    let bml_element_new = construct_bookmarklet(kbdid, kbd.LanguageCode, kbd.Name, label);
```
Comment (darcywong00) [inline cdn/dev/js/kmwlive.js:451]: Do we need to use `var` instead of `let`? See previous #49

**Review by mcdurdin — APPROVED** [review]
Apart from @darcywong00's question, and any changes required as an outcome of my review of https://github.com/keymanapp/r.keymanweb.com/pull/7, this LGTM.

File: inc/head.php [hunk 2]
```diff
@@ -207,7 +207,13 @@ function loadKeyboardFromHash() {
...
     if(typeof addKeyboards == 'function') {
       addKeyboards();
-    }
+    } <?php if ($site_suffix == ".local") {
+      echo 'else {
+      console.warn("-- Fallback:  not using the server\'s cached keyboard set! --");
+      // Server caching may not be active in local dev instances of the server.
+      keyman.addKeyboards();
+    }';
+    } ?>
```
Comment (jahorton) [inline inc/head.php:216]: Kept having to manually change this for testing, so I figured I'd throw it in for `.local` mode only.

Comment (mcdurdin) [reply inc/head.php:216]: OK I guess? Not sure I'm understanding why this is happening?

Comment (jahorton) [reply inc/head.php:216]: No keyboards show when I use my local version of the page unless I make this change.  I noticed that the `addKeyboards` called by the `if` portion refers to caching query results, which my local setup doesn't seem to facilitate.

Comment (mcdurdin) [reply inc/head.php:216]: I feel like there is another problem happening here for which this is a symptom. It's probably not all that important though so go ahead with this :grin:

**Pull request merged**